### PR TITLE
Infrastructure for GemStone-specific eun.sh args

### PR DESCRIPTION
**Pull request by dalehenrich**
- Remove the --gs-* args that don't apply quite yet
- Partial fix for Issue #102 - fix typos for the `--clean` messages
- changes needed to make the `--gs-*` args code fuctional ...
- fix a GemStone-only bug in `--clear`

### CODE REVIEW

**Comment by fniephaus** [comment]
I will need to look into this tomorrow again. It's already pretty late here :)

**Comment by dalehenrich** [comment]
no hurry ...

File: run.sh [hunk 1]
```diff
@@ -270,6 +272,7 @@ clean_up() {
     fi
     if is_dir "${SMALLTALK_CI_BUILD_BASE}"; then
       print_info "  ${SMALLTALK_CI_BUILD_BASE}"
+      chmod -fR +w "${SMALLTALK_CI_BUILD_BASE}"
```
Comment (fniephaus) [inline run.sh]: Why is this necessary? You are not doing the same with `${SMALLTALK_CI_CACHE}`.

**Comment by fniephaus** [comment]
Could you check out 1667ebbbebcd21b1a5dccf817dd1742a3d6d8a22 and verify that `--gs-*` arguments are still working? I quickly added two dummy options and they are working for me:

image

File: run.sh (hunk 1 above)
Comment (dalehenrich) [inline run.sh]: The GemStone download is chock full of read only files and the `chmod` is required before `rm`ing the `_build` directory. There are no read only GemStone artifacts in the cache directory so it isn't required ...

**Comment by dalehenrich** [comment]
If they work for you then I'm fine ... I'll test your changes after I merge master again ... I will  be adding some real options shortly, so I'll test for myself at that point in time ...

File: run.sh (hunk 1 above)
Comment (fniephaus) [inline run.sh]: Cool, I didn't know. Adding a comment...
